.github/actions/kibana-ftr/README.md

```diff
@@ -0,0 +1,76 @@
+# Run Kibana FTR
+
+This GitHub Action runs Kibana tests using the Functional Test Runner ([FTR](https://www.elastic.co/guide/en/kibana/current/development-tests.html#development-functional-tests)).
+
+## Inputs
+
+| Name                 | Description                                    | Required | Default                                                      |
+|----------------------|------------------------------------------------|----------|--------------------------------------------------------------|
+| `test_kibana_url`    | URL for the Kibana instance to test            | true     |                                                              |
+| `test_es_url`        | URL for the Elasticsearch instance to test     | true     |                                                              |
+| `es_version`         | Version of Elasticsearch to test against       | true     |                                                              |
+| `ref_commit_sha`     | Kibana PR commit sha                           | false    | `main`                                                       |
+| `test_browser_headless` | Whether to run tests in headless mode        | false    | `1`                                                          |
+| `test_cloud`         | Whether to enable cloud testing                | false    | `1`                                                          |
+| `es_security_enabled` | Whether Elasticsearch security is enabled     | false    | `1`                                                          |
+| `node_version_file`  | Path to the Node.js version file               | false    | `package.json`                                               |
+| `config`             | Path to the test configuration file            | false    | `x-pack/test/cloud_security_posture_functional/config.cloud.ts` |
+
+## Usage
+
+```yaml
+name: Kibana UI Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref_commit_sha:
+        type: string
+        description: |
+          Kibana PR commit sha
+        required: false
+
+jobs:
+  run-kibana-tests:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+
+      - name: Run Kibana Tests Action
+        uses: ./.github/actions/kibana-ftr
+        with:
+          ref_commit_sha: ${{ github.event.inputs.ref_commit_sha || 'main' }}
+          test_kibana_url: ${{ secrets.TEST_KIBANA_URL }} # required
+          test_es_url: ${{ secrets.TEST_ES_URL }} # required
+          es_version: 8.15.0-SNAPSHOT # required
+          test_browser_headless: '1'
```

Comment:
why are we using `'1'` as a boolean?

Reply:
This originates from the FTR runner, where setting the environment variable to '1' activates headless mode. However, as per @orouz's suggestion, I'll eliminate this input variable. Since all tests are executed in headless mode on the CI, no further adjustments are necessary.